services/core-api/app/api/notice_of_departure/models/notice_of_departure_contact.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy import FetchedValue
+from sqlalchemy.dialects.postgresql import UUID
+
+from app.api.utils.models_mixins import SoftDeleteMixin, AuditMixin, Base
+from app.extensions import db
+
+
+class NoticeOfDepartureContact(SoftDeleteMixin, AuditMixin, Base):
+    __tablename__ = 'notice_of_departure_contact'
+
+    nod_contact_guid = db.Column(
+        UUID(as_uuid=True), primary_key=True, server_default=FetchedValue())
+    nod_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('notice_of_departure.nod_guid'))
+    first_name = db.Column(db.String(100), nullable=False)
```

Comment:
Frontend accepts 200, but we are limiting to 100 in the model
```suggestion
    first_name = db.Column(db.String(200), nullable=False)
```